py/desiutil/bitmask.py

```diff
@@ -100,6 +100,52 @@ def __str__(self):
     # def __repr__(self):
     #     return "_MaskBit(name='{0.name}', bitnum={0.bitnum:d}, comment='{0.comment}')".format(self)
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        """When working with Numpy functions and objects, replace the input
+        ``_MaskBit`` object with its ``.mask`` attribute.
+        """
```

Comment:
To help future maintainers, let's expand these comments to explain more about why this is necessary and provide more context for people who haven't read and understood `__array_ufunc__` documentation.  At minimum reference issue #218 and numpy docs for details; ideally also provide examples of what this function enables / what wouldn't work without it and that this was needed due to changes in numpy 1.x to 2.x.

Reply:
doc string is now expanded.